graphkb/genes.py

```diff
@@ -161,3 +162,200 @@ def get_genes_from_variant_types(
         ),
     )
     return result
+
+
+def get_preferred_gene_name(conn: GraphKBConnection, gene_name: str, source: str = '#39:5') -> str:
+    """Preferred gene symbol of a gene or transcript.
+
+    Args:
+        gene_name: the gene name to search features by
+        ignore_cache (bool, optional): bypass the cache to always force a new request
+        source: id of the preferred gene symbol source
+    Returns:
+        preferred displayName symbol.
+
+    Example:
+        return KRAS for get_preferred_gene_name(conn, 'NM_033360')
+        return KRAS for get_preferred_gene_name(conn, 'ENSG00000133703.11')
+    """
+    CHROMOSOMES = [f"chr{i}" for i in range(1, 24)] + ['chrX', 'chrY']
```

Comment:
might make sense to move this to `constants.py`

Reply:
Also `x`, `y`, and `mt` are references used in GraphKB. What happens if one of those is passed?

Reply:
> Also x, y, and mt are references used in GraphKB. What happens if one of those is passed?

Currently, they would be mistaken for gene names.

I think its a good idea to move this to constants.py.  Will do.